src/main/scala/com/cognite/sdk/scala/v1/Client.scala

```diff
@@ -48,6 +48,34 @@ final case class RequestSession[F[_]: Monad](
         .parseResponseIf(_ => true)
     ).send()(sttpBackend, implicitly).map(_.unsafeBody)
 
+  def postProtobuf[R](r: RequestT[Empty, String, Nothing] => RequestT[Id, R, Nothing]): F[R] =
```

Comment:
Modify `sendCdf` to instead have additional parameters with `contentType: String = "application/json", accept: String = "application/json"`